apple-pie/styles/style.css

```
@@ -0,0 +1,53 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+body{
+    font-family: 'Rubik', sans-serif;
+    background-color: white;
+}
+
+header{
+    background-image:url(/apple-pie/images/apple-pie.jpg);
+    background-position: 0 0;
+    background-repeat: no-repeat;
+    background-size:cover;
+    width: 100%;
+    height: 1000px;
+    
+
+}
+header h1{
+    color: white;
+    font-size: 7em;
+    text-align: center;
+    margin: 0;
+    display: flex;
+   padding: 370px 600px;
+    width: 1000px;
+}
+.info{
+    font-size: 2em;
+    text-align: center;
+    padding: 100px 100px 50px 100px;
+    border-bottom: grey solid;
+    
+}
+.ingredientes{
+    font-size: 1.5em;
+    border-bottom: grey solid;
+    padding-top: 50px;
+}
+.ingredientes h2{
+    font-size: 1.6em;
+    padding-bottom: 30px;
+}
+.ingredientes img{
+   padding: 50px 600px;
+}
+.directions{
+    font-size: 1.5em;
+    padding-top: 50px;
+    
+}
+.directions h2{
+    font-size: 1.6em;
+    padding-bottom: 30px;
+}
```

Con flexbox alinea todos los elementos que contiene y modifique el estilo de la lista para que coincidiera ```suggestion
}
.container{
    display: flex;
    align-items: center;
    justify-content: center;
    flex-direction: column;
    width: 100vw; 
    margin: auto;
}

img{
    width: 500px;
}
ul>li{
    list-style: "-";
}
```